src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -2570,18 +2569,16 @@ void GenericThreadStackManagerImpl_OpenThread<ImplClass>::OnDnsBrowseResult(otEr
 
         VerifyOrExit(error == CHIP_NO_ERROR, );
 
-        DnsResult * dnsResult = Platform::New<DnsResult>(aContext, MapOpenThreadError(aError));
+        DnsResult * dnsResult = Platform::New<DnsResult>(aContext, CHIP_NO_ERROR);
```

Comment:
previous code was doing a map of aError. New code just uses CHIP_NO_ERROR. was the old code a bug? 

Same question for updates below.

Reply:
I now see above `VerifyOrExit(aError == OT_ERROR_NONE, error = MapOpenThreadError(aError));` ... so old code seems indeed awkward.